manuscript/110_Constrained_Non_Determinism.md

```diff
@@ -34,7 +34,7 @@ ShouldCreateBackupFileNameContainingPassedHostName()
 }
 ```
 
-Although the relationship can be guessed quite easily, remember it is just an example. Also, seeing code like that makes me ask questions like: is the �backup\_" prefix always applied? What if I�actually pass �backup\_" instead of �MY\_HOST\_NAME"? Will the name be "backup\_backup\_.zip", or "backup\_.zip"? Also, is this object responsible for any validation of passed string?
```

Comment:
How about changing "actually" to "already"? (this would probably need some rearrangement of the sentence..?)